src/common/hooks/useFilters.tsx

```diff
@@ -61,6 +61,11 @@ export default function useFilters({
     filter = hasPagination ? { page: 1, limit, ...filter } : filter;
     setQueryParams(Object.assign({}, qParams, filter), { replace: true });
   };
+  const updateSerchByField = (filter: FilterState) => {
+    filter = hasPagination ? { page: 1, limit, ...filter } : filter;
+    console.log(filter);
+    setQueryParams(Object.assign({}, filter), { replace: true });
+  };
```

Comment:
i don't see a reason why `updateQuery` is not sufficient? Why duplicate the same implementation with another name?

Reply:
@i0am0arunava when you are resolving something without addressing add the reason and let the author resolve it